content/en/blog/trusted-boot.md

```diff
@@ -0,0 +1,155 @@
+And then, we have Measured Boot. Picture this as Alice keeping a super detailed logbook. She writes down everything that goes on in her house – who comes in, what they do, you name it. If anything odd pops up, she’s on it. Measured Boot does something similar for your computer. It watches over the startup process, takes notes, and makes sure everything's running just like it should, no funny business.
+
+Putting all these together, you’ve got Trusted Boot, which is like Alice's ultimate home security system – the secret-code lock, the smart doorbell, and the detailed logbook, all working together. We’re going to unpack how these nifty tools keep your computer as secure as Alice’s treasure chest, making sure your digital world is safe every time you power up.
+
+
+## Challenges and Solutions in Implementing FDE on Linux
+
+![DALL·E](https://github.com/kairos-io/kairos-docs/assets/2420543/aee89c19-a5d1-4b75-8b05-bbeef4e2b47e)
+
+
+While FDE is a standard feature in various operating systems, its implementation in Linux presents unique challenges. The initial boot stage, including the kernel and initial file system setup, cannot be fully encrypted, leaving a portion of the system exposed. However, through careful measures, this unencrypted segment can be protected against tampering, thus securing the system's boot process and the encrypted data it accesses.
+
+The Kairos project introduces "Unified Kernel Images" (UKI) to enhance boot security. UKI files are single, fat binaries that contain the OS and necessary boot components in a single, verified file. This approach simplifies the boot process, allowing for the entire system to be booted securely and efficiently. By leveraging EFI files that can be signed and verified through Secure Boot and measured as a single entity, UKI files represent a significant advancement in boot security.
+
+Central to the Trusted Boot mechanism is the Trusted Platform Module (TPM), a dedicated hardware(also emulated by firmware) component designed for secure cryptographic operations. TPM chips play a critical role in storing encryption keys and measurements securely, enabling the system to verify boot integrity and encrypt user data effectively.
+
+
+## From Grub to systemd-boot
+
+![DALL·E 2024-01-18 18 22 47 - Create a vintage-style digital artwork that visualizes the concept of modernizing technology infrastructure  The central focus should be a retro-futur](https://github.com/kairos-io/kairos-docs/assets/2420543/df3bce6f-4e88-4edd-83b2-ba2d333d9e6f)
+
+Trusted Boot is deeply integrated in systemd and currently under active development. Systemd is renowned for being more than a mere init system. It includes a range of tools like systemd-measure and ukify, which, though compatible with various bootloaders, synergize exceptionally well with systemd-boot. This makes a compelling case for choosing systemd, and particularly systemd-boot.
```

Comment:
I think this paragraph could be improved if the line of thought went into. There are other bootloaders, they work well on linux, but systemd-boot is prefered for these reasons. No need to mention that systemd is more than an init system.